.rubocop.yml

```diff
@@ -50,3 +52,9 @@ Style/HashTransformKeys:
 Style/HashTransformValues:
   Enabled: true
 
+Style/NumericPredicate:
+  Enabled: false
+
+# syntax is non obvious to a non ruby developer
```

Comment:
Hmm the double negative could use a bit of a reword. Something along these lines?
```suggestion
# syntax is not common outside of the ruby world, try to reduce the opinionated back and forth
```

Reply:
👍 Seems fine to me, agree that the double negative is weird here.